BUILD.bazel

```diff
@@ -41,6 +41,7 @@ cc_library(
         "stout/collect.h",
         "stout/filter.h",
         "stout/take.h",
+        "stout/range.h",
```

Comment:
Sorry but you'll need to update this because we broke everything into multiple sub-libraries.